modules/migration-configuring-mcg.adoc

```diff
@@ -37,12 +37,15 @@ endif::[]
 
 .Procedure
 ifdef::installing-3-4,installing-mtc[]
-* Obtain the S3 endpoint, `AWS_ACCESS_KEY_ID`, and `AWS_SECRET_ACCESS_KEY` by running the link:https://docs.redhat.com/en/documentation/red_hat_openshift_data_foundation/4.15/html/managing_hybrid_and_multicloud_resources/accessing-the-multicloud-object-gateway-with-your-applications_rhodf#accessing-the-Multicloud-object-gateway-from-the-terminal_rhodf[`describe` command] on the `NooBaa` custom resource.
+* Obtain the S3 endpoint, `AWS_ACCESS_KEY_ID`, and the `AWS_SECRET_ACCESS_KEY` value by running the `oc describe` command for the `NooBaa` CR.
 +
 You use these credentials to add MCG as a replication repository.
 endif::[]
 ifdef::installing-oadp-mcg[]
-. Obtain the S3 endpoint, `AWS_ACCESS_KEY_ID`, and `AWS_SECRET_ACCESS_KEY` by running the link:https://docs.redhat.com/en/documentation//red_hat_openshift_data_foundation/4.15/html/managing_hybrid_and_multicloud_resources/accessing-the-multicloud-object-gateway-with-your-applications_rhodf#accessing-the-Multicloud-object-gateway-from-the-terminal_rhodf[`describe` command] on the `NooBaa` custom resource.
+. Create a MCG bucket. For more information, see link:https://docs.redhat.com/en/documentation/red_hat_openshift_data_foundation/4.18/html-single/managing_hybrid_and_multicloud_resources/index[Managing hybrid and multicloud resources].
```

Comment:
This is confusing, but here we'd use "an" instead of "a" because the "M" in "MCG" is vowel-sending (as in "empty"). See https://www.ibm.com/docs/en/ibm-style?topic=grammar-articles for more information if warranted. 

```suggestion
. Create an MCG bucket. For more information, see link:https://docs.redhat.com/en/documentation/red_hat_openshift_data_foundation/4.18/html-single/managing_hybrid_and_multicloud_resources/index[Managing hybrid and multicloud resources].
```